README

```diff
@@ -1,4 +1,4 @@
-This is the code of the Dancer's website
+This is the code of a Dancer's Website
```

Comment:
The original line wasn't right, but the change isn't quite right either. Should probably drop a/the altogether.

Reply:
Yeah, either "the code of the Dancer website" or "the code of Dancer's website" would be more correct; I'd go with the former.